cmd/watch.go

```diff
@@ -102,6 +109,16 @@ func handleWatchEvent(client kit.ThemeClient, asset kit.Asset, event kit.EventTy
 		blue(asset.Key),
 	)
 	if err := perform(client, asset, event, nil); err != nil {
+		notificationMessage("There is a error in your templates check your logs", "Watch", "")
 		stdErr.Printf("[%s] %s", green(client.Config.Environment), err)
 	}
 }
+
+func notificationMessage(message string, title string, iconPath string) {
+	notify = notificator.New(notificator.Options{
```

Comment:
I would prefer to have this disabled by default and enabled using a command line flag. I don't think the default behavior of any command line tool should leave that domain unless given permission explicitly. In this case by a command line flag.

Reply:
In [Laravel Mix](https://github.com/JeffreyWay/laravel-mix), which is where I got the idea they have it enabled by default but we can make into flag that is not an issue.